h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -273,6 +281,18 @@ public boolean onExceptionalCompletion(Throwable ex, CountedCompleter caller) {
     }
 
     public abstract void computeImpl();
+
+    public final void computeParameters() {
+      M model = _result.get();
+      if (model != null) {
+        //set input parameters
+        model.setInputParms(_input_parms);
+        //set _fold_assignment param to null in case it was entered as AUTO and it wasn't set during the training
```

Comment:
I can't remember other reason I did this like this than because fold_assignment didn't have null enabled as input value. Now as it has it enabled I can remove this and set fold assignment to null in EffectiveParametersUtils#initFoldAssignment() in else {...}. Logically it belongs there but it wasn't possible with null disabled.

Reply:
fixed